### PR TITLE
ADB 1.6 Announcement

### PR DESCRIPTION
**Pull request by bexelbie**
First DRAFT

Don't merge yet - giving this a slightly wider visibility

@jzb @navidshaikh

### CODE REVIEW

**Comment by jzb** [comment]
If this is WIP, can we schedule for Monday? I don't want to post stuff on Fridays for major releases if we can help it.

**Comment by bexelbie** [comment]
@jzb +1 for Monday
Also, I'd like @navidshaikh to review it first

File: source/blog/2016-01-25-ADB-1.6-announcement.html.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  * [Mesos Marathon](https://mesosphere.github.io/marathon/) as a supported container orchestrator
+  * A new delivery location in HashiCorp's Atlas box catalog, [projectatomic/adb](https://atlas.hashicorp.com/projectatomic/boxes/adb)
+  * Updated Documentation, a new Maintainer and [more](https://github.com/projectatomic/adb-atomic-developer-bundle/releases/tag/v1.6.0)
+
+The ADB provides a container development ecosystem that ensures that developers are able to work with Linux containers even if their base operating system does not support it.
+
+READMORE
+
+The ADB provides a quick and easy way to setup multi-container applications that take advantage of management and/or orchestration solutions, such as OpenShift or [Kubernetes](http://kubernetes.io/). The ADB also supports [Nulecule](http://www.projectatomic.io/docs/nulecule/) and [Atomic App](http://www.projectatomic.io/docs/atomicapp/). These projects provide a structure for packaging, distributing, and deploying multi-container applications.
```
Comment (navidshaikh) [inline source/blog/2016-01-25-ADB-1.6-announcement.html.md]: a hyperlink to OpenShift should also help, as we are hyperlinking Kubernetes or Nulecule, etc.

[OpenShift] (https://www.openshift.com/)

Comment (navidshaikh) [inline source/blog/2016-01-25-ADB-1.6-announcement.html.md]: Rest LGTM!

**Comment by bexelbie** [comment]
@navidshaikh comment implemented

@jzb @jberkus this is ready to go - however we are in a queue for files to be uploaded to cloud.centos.org.  Therefore, I will update this again when the files are live.

**Comment by jzb** [comment]
Thanks, still planning on Monday barring any counter-info. @bproffitt this is ready for editing, I believe.

**Comment by bexelbie** [comment]
@jberkus , pending any comments by @bproffitt, this is ready to go.

@jzb @navidshaikh @LalatenduMohanty fyi

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**